core/src/main/java/org/apache/carbondata/core/datamap/DataMapStoreManager.java

```diff
@@ -269,26 +275,23 @@ public synchronized void unRegisterDataMapCatalog(DataMapSchema dataMapSchema) {
    * @param providerName
    * @return
    */
-  public synchronized DataMapCatalog getDataMapCatalog(
-      DataMapProvider dataMapProvider,
-      String providerName) throws IOException {
+  public synchronized DataMapCatalog getDataMapCatalog(DataMapProvider dataMapProvider,
```

Comment:
move `DataMapProvider dataMapProvider,` to next line

Reply:
done